C4G.Unity/Assets/C4G/Editor/Scripts/C4GSettings.cs

```diff
@@ -38,7 +43,18 @@ public string SerializedConfigsFolderFullPath
 
         public bool IsGeneratedCodeFolderValid => !string.IsNullOrEmpty(_generatedCodeFolderPath) &&
                                                   Directory.Exists(GeneratedCodeFolderFullPath);
+
         public bool IsSerializedConfigsFolderValid => !string.IsNullOrEmpty(_generatedCodeFolderPath) &&
-                                                  Directory.Exists(SerializedConfigsFolderFullPath);
+                                                      Directory.Exists(SerializedConfigsFolderFullPath);
+
+        public IReadOnlyDictionary<string, SheetParserBase> SheetConfigurations
+        {
+            get
+            {
+                return _sheets
```

Comment:
[Critical] If multiple `SheetEntry` objects have the same `sheetName`, `ToDictionary()` will throw an `ArgumentException`.
Let's add validation or use a safer approach:
`.GroupBy(e => e.sheetName).ToDictionary(g => g.Key, g => g.First().parserBase)`
Or better yet, validate in the editor to prevent duplicates 😃